docs/csharp/quick-starts/hello-world.yml

```diff
@@ -0,0 +1,171 @@
+### YamlMime:YamlDocument
+documentType: Tutorial
+title: Hello C#
+metadata:
+  title:  Hello C#. Your first introduction to the C# language.
+  description: This tutorial teaches you C# interactively, using your browser to write C# and see the results of compiling and running your code. It contains a series of lessons that begin with a "Hello World" program. These lessons teach you the fundamentals of the C# language.
+  audience: Developer
+  level: Beginner
+  nextTutorialHref: working-with-numbers
+  nextTutorialTitle: Working with numbers in C#
+  displayType: two-column
+  interactive: csharp
+items:
+- durationInMinutes: 1
+  content: |
+    This tutorial teaches you C# interactively, using your browser to write C# and see the results of compiling and running your code. It contains a series of lessons that begin with a "Hello World" program. These lessons teach you the fundamentals of the C# language.
+- title: Run your first c# program
+  durationInMinutes: 2
+  content: |
+    Run the following code in the interactive window (type the following line in the window on the right and click the 'Run' button):
```

Comment:
Instead of putting in parentheses, I suggest putting in a new sentence.
For example:
Run the following code in the interactive window. To do that, type (or maybe copy?) the following line...